modules/consensus/accept_test.go

```diff
@@ -1349,7 +1351,7 @@ func (cst *consensusSetTester) testPaymentChannelBlocks() error {
 		// txn needs to be fully custom. To get a custom txn, manually create an
 		// address and then use the wallet to fund that address.
 		channelSize := types.NewCurrency64(10e3)
-		channelFundingSK, channelFundingPK, err := crypto.GenerateSignatureKeys()
```

Comment:
ok. It's not how the go standard library works, but it's good enough for me. I see the fear, though there's not much room to introduce error when you've just made a passthrough function.